src/modules/groupAccess.ts

```diff
@@ -1,11 +1,36 @@
+import { prisma } from '../utils/prisma';
 import { AccessCheckResult, allowed, notAllowed } from '../utils/access';
 import { SessionUser } from '../utils/auth';
 
 import { tr } from './modules.i18n';
 
+const checkGroupAdmin = async (groupId: string, userId: string) => {
+    const admins = await prisma.$queryRaw<Array<{ count: number }>>`
+    SELECT count(*) FROM public."GroupAdmin"
+    WHERE "userId" = ${userId}
+    AND "groupId" in (
+        WITH RECURSIVE rectree AS (
+            SELECT *
```

Comment:
could probably select just the id, we don't need actual data here. Don't know if it affects performance or not

Reply:
@9teen90nine help,pls. What do you think about it?

Reply:
you will need to select parentId also and that will drop all the performance gains because it's a full record lookup